TI/About.html

```diff
@@ -0,0 +1,188 @@
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<head>
+    <title>About</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
+    integrity="sha384-fJjVfl3If3sE2q3+3kD1RxTIsp/7sz+b65E+Gbh/aIuNRSZ6DTlQz9XW2IaUpiSN"
+    crossorigin="anonymous">
+</head>
+<header>
+  <nav class="navbar navbar-expand-lg my-custom-navbar">

```

Comment:
Menu is not visible on mobile. Also, the page alignment is not good on mobile.

<img width="448" alt="image" src="https://user-images.githubusercontent.com/6362750/235317708-f8c896f6-786f-4379-a2cb-4be694a511f5.png">